app/misc/git.ts

```diff
@@ -495,6 +495,37 @@ function createBranch() {
   }
 }
 
+// search for tags
+function searchTag() {
+  Git.Repository.open(repoFullPath)
+    .then(function (repo) {
+      repo.getCurrentBranch()
+        .then(function () {
+          // grab the list of references - these could be branches or tags
+          return repo.getReferences(Git.Reference.TYPE.LISTALL);
+        }).then(function (refList) {
+          refList.sort();
+          for (let i = 0; i < refList.length; i++) {
+
+            // strip name for readability
+            let refName = refList[i].name().split("/")[refList[i].name().split("/").length - 1];
```

Comment:
Logically you can move this into the if statement?

Also, can you state what format the name is, please?
ie
name is "/blah/blah/blah/name" and we want to extract "name"?

Reply:
refName is the readable, string format of the tag name. The string returned from nodegit needs to be stripped before it can be used. 

Which if statement are you referring to?

Reply:
It's an optimization. You are doing the parsing of the name BEFORE checking if it is a tag. You should check if it is a tag before parsing the name. Hence move let refName =... into the if (refList[i].isTag()) {